rca/people/tests/test_models.py

```diff
@@ -28,3 +30,39 @@ def test_page_count_rules(self):
         )
         # A second person listing page should not be creatable
         self.assertFalse(StudentIndexPage.can_create_at(self.home_page))
+
+
+class TestStudentPage(WagtailPageTests):
+    def setUp(self):
+        self.home_page = HomePage.objects.first()
+        self.user = self.login()
+        self.home_page.add_child(
+            instance=StudentIndexPage(
+                title="Students", slug="students", introduction="Students"
+            )
+        )
+        self.student_index = StudentIndexPage.objects.first()
+        self.student = User.objects.create_user(
+            username="danascully",
+            first_name="dana",
+            last_name="Scully",
+            email="ds@fbi.com",
+            password="1234",
+        )
+        self.student = User.objects.get(username="danascully")
+        self.collection = CollectionFactory(name="Student: dana scully")
+
+    def test_creating_creates_permision_group(self):
+        self.student_index.add_child(
+            instance=StudentPage(
+                title="Dana Scully",
+                slug="a-student",
+                first_name="dana",
+                last_name="student",
+                student_user_image_collection=self.collection,
+                student_user_account=self.student
+                # {'student_user_account': ['user instance with id 3 does not exist.']}
+                # I do not understand why, I can see it exists!
+            )
+        )
```

Comment:
As the comment states here, I've no idea why the user isn't available in the test run.
In the debug here https://github.com/torchbox/rca-wagtail-2019/blob/60a31e25a0348b3729d3c71c9da183b3bb055763/rca/people/models.py#L887 on saving the page via the admin, I see the user in the logs, but in the test run this prints None